tests/BaseTestCase.php

```diff
@@ -26,13 +26,24 @@ public static function setUpBeforeClass()
 
         self::$dic['loggerClass'] = $logClass;
 
-        self::$dic['configuration'] = function($c) {
-            return FF::getInstance(
-                'Core\XmlConfiguration',
-                RESOURCES_DIR.DS.'config.xml',
-                'test'
-            );
-        };
+        if (strpos(static::class, 'ArrayConfiguration')) {
```

Comment:
This (obviously) is a hack, needed because the 'configuration' key is set in setUpBeforeClass() and can't be overwritten later.